dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogTest.kt

```diff
@@ -249,6 +250,24 @@ internal class DatadogTest {
             .containsExactly(ConnectionSpec.RESTRICTED_TLS)
     }
 
+    @Test
+    @TestTargetApi(Build.VERSION_CODES.KITKAT)
+    fun `add compatibility network policy for https endpoints on 19+`(forge: Forge) {
+        val endpoint = forge.aStringMatching("https://[a-z]+\\.[a-z]{3}")
+
+        Datadog.initialize(mockAppContext, fakeToken, endpoint)
+
+        val uploader: DataOkHttpUploader = Datadog.javaClass.getStaticValue("uploader")
+        val okHttpClient: OkHttpClient = uploader.getFieldValue("client")
+
+        assertThat(okHttpClient.protocols())
+            .containsExactly(Protocol.HTTP_2, Protocol.HTTP_1_1)
+        assertThat(okHttpClient.callTimeoutMillis())
+            .isEqualTo(Datadog.NETWORK_TIMEOUT_MS.toInt())
+        assertThat(okHttpClient.connectionSpecs())
+            .containsExactly(ConnectionSpec.COMPATIBLE_TLS)
```

Comment:
Is there any reason here you are asserting the `ConnectionSpec.COMPATIBLE_TLS` but in the code you are returning `ConnectionSpec.MODERN_TLS` ?

Reply:
Indeed, that's an error

Reply:
Done :)